packages/wrangler/src/deploy/deploy.ts

```diff
@@ -433,6 +436,14 @@ See https://developers.cloudflare.com/workers/platform/compatibility-dates for m
 
 	const { format } = props.entry;
 
+	if (!props.dispatchNamespace && prod && accountId && scriptName) {
+		const yes = await confirmLatestDeploymentOverwrite(accountId, scriptName);
+		if (!yes) {
+			cancel("Aborting deploy...");
```

Comment:
This is mixing new style (c3/cli) with a command using the old style (wrangler). You can just `logger.error` in this command but I don't think it's a big deal to leave as-is (nit)

Reply:
Leaving this as-is with `cancel` because `logger.error` seems to write to a log file somewhere? Not what we want in this case.

Reply:
all logs are meant to go to the log file for debug-ability but no big deal to leave this